### ISSUE TITLE
Remove references to SMIL

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria#973]
Remove references to SMIL from:
- section role's Related Concepts: SMIL par
- Normative references [SMIL3]
- aria-expanded Related Concepts should be taken care of in pull 972
